src/Okta.Idx.Sdk/IAuthenticator.cs

```diff
@@ -36,5 +36,15 @@ public interface IAuthenticator
         /// Gets or sets the authenticator method types if applicable.
         /// </summary>
         IList<AuthenticatorMethodType> MethodTypes { get; set; }
+
+        /// <summary>
+        /// Gets or sets the QrCode
+        /// </summary>
+        IQrCode QrCode { get; set; }
```

Comment:
Can we add more info in the comments about when these props are available? i.e "Gets or sets the QrCode. This is applicable to OTP authenticators such as Google Authenticator".